### ISSUE TITLE
Prioritize module with higher update rates when internal and external module are active

### ISSUE THREAD

**Issue by mha1** [EdgeTX/edgetx#2919]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

The current implementation always prioritizes the internal module when setting the update rate for the mixer scheduler. This forces the external module to the same update rate. Consider the internal module being a MPM, the external module being a ELRS module set at 500Hz update rate. As the MPM is updated at a 7000us period (143Hz) parallel operation of both modules forces the ELRS module to 143Hz although it is set to 500Hz. Most likely will make an internal ELRS module set at 500Hz make the parallel use of an external MPM impossible as the external MPM will now be updated at 500Hz (MPM stops working at much higher rates than 143Hz).

### Describe the solution you'd like

Due to the way the mixer scheduler is driven (single timer interrupt) only one module can drive the mixer schedule with its requested update rate and synced by using the driving modules period and lag data. Let's call this (the faster module) module the master module. The slower module (slave module) could be served using a integer fraction of the masters update rate which should be as close to the slaves requested rate but not exceeding it. It must be said that the salve module then runs in a non-synced way,

Example 1: ELRS set at 333Hz, MPM uses 143 Hz
old approach: ELRS runs at non-synced 143Hz, MPM runs at synced 143Hz
new approach: ELRS runs as master at 333Hz synced, MPM runs as salve at non-synced 111Hz (divider 3)

Example 2: ELRS set at 100Hz, MPM uses 143 Hz
old aproach: old approach: ELRS runs at non-synced 143Hz, MPM runs at synced 143Hz
new approach: ELRS runs as salve atnon-synced 71Hz (divider 2), MPM runs at master at synced 143Hz

To summarize: implement logic to identify the module with the faster update rate, make it the synced master module and run the other module as a slave module non-synced at a integer fraction of the master modules update rate. Please note, it won't matter if the master module is internal or external. Calculation is not based on where the module is installed but purely on update rate.
